rosidl_typesupport_opensplice_cpp/include/rosidl_typesupport_opensplice_cpp/requester.hpp

```diff
@@ -282,6 +285,46 @@ class Requester
     return estr;
   }
 
+  /// Return NULL on success, otherwise an error string.
+  const char *
+  server_is_available(
+    const rmw_node_t * node, bool * is_available,
+    rmw_ret_t (* count_publishers)(const rmw_node_t *, const char *, size_t *),
+    rmw_ret_t (* count_subscribers)(const rmw_node_t *, const char *, size_t *)
+  ) noexcept
+  {
+    if (!is_available) {
+      return "argument is_available is null";
```

Comment:
No, these functions are strange like that. See: https://github.com/ros2/rmw_opensplice/blob/master/rosidl_typesupport_opensplice_cpp/include/rosidl_typesupport_opensplice_cpp/service_type_support.h#L62-L66